python/tests/test_reports.py

```diff
@@ -48,6 +48,13 @@ def test_get_spikes_from_population(self):
 
         self.assertEqual(len(self.test_obj['All'].get(node_ids=[])), 0)
 
+        dict_data = self.test_obj['All'].get_dict()
+        self.assertTrue((dict_data["node_ids"] == np.asarray([5, 2, 3, 2, 3])).all())
+        self.assertTrue((dict_data["timestamps"] == np.asarray([0.1, 0.2, 0.3, 0.7, 1.3])).all())
+        dict_data_filtered = self.test_obj['All'].get_dict(node_ids=[5, 3], tstart=0.2)
+        self.assertTrue((dict_data_filtered["node_ids"] == np.asarray([3, 3])).all())
+        self.assertTrue((dict_data_filtered["timestamps"] == np.asarray([0.3, 1.3])).all())
+
```

Comment:
could you add tests for just using tstart/tstop?

I happend to run this:
```
import libsonata
sr=libsonata.SpikeReader('spikes.h5')
sr['All'].get_dict(tstart=0)
```

And got:
```
RuntimeError: bad optional access
```